packages/account/src/Sections/Profile/PersonalDetails/personal-details-form.tsx

```diff
@@ -617,7 +608,7 @@ const PersonalDetailsForm = observer(() => {
                                                 </div>
                                                 {isPhoneNumberVerificationEnabled && (
                                                     <VerifyButton
-                                                        is_dynamic_fa_enabled={isDynamicFAEnabled}
+                                                        is_dynamic_fa_enabled={true}
```

Comment:
```suggestion
                                                        is_dynamic_fa_enabled
```

Reply:
Remove `is_dynamic_fa_enabled ` because it should be `true` by default